ui.frontend/src/view/FormFileInputWidget.js

```diff
@@ -63,11 +63,21 @@
         }
 
         attachEventHandlers(widget, model) {
+            widget.addEventListener('click', (e)=> {
```

Comment:
What are these changes to do with allow attaching duplicate files? Why is a click listener required here, its not clear.

Reply:
Moved the clearing Widget value logic in handleChange function